20230727/s12-1690491686694/README.md

```diff
@@ -0,0 +1,66 @@
+# Scenario 12 - ALS Baseline with Sims-only, Disabled JSON.stringify. ALS v14.2.3 + ALS Scale 2.
```

Comment:
Can we have a consistent naming convention for all the scenarios? With high level scenario description, ALS version, ALS scale, K6 VU, Comments across all the scenarios?